services/migration_service/migration_files/20210202145952_add_runs_idx_ts_epoch_flow_id.sql

```diff
@@ -0,0 +1,60 @@
+-- +goose Up
+-- +goose StatementBegin
+SELECT 'up SQL query';
+
+-- Others
+
+CREATE INDEX runs_v3_idx_ts_epoch ON runs_v3 (ts_epoch);
+
+CREATE INDEX runs_v3_idx_gin_tags ON runs_v3 USING gin (tags, system_tags);
+CREATE INDEX runs_v3_idx_gin_tags_combined ON runs_v3 USING gin ((tags || system_tags));
+
+-- flow_id + ts_epoch
+
+CREATE INDEX runs_v3_idx_flow_id_asc_ts_epoch_desc ON runs_v3 (flow_id ASC, ts_epoch DESC);
+CREATE INDEX runs_v3_idx_flow_id_desc_ts_epoch_desc ON runs_v3 (flow_id DESC, ts_epoch DESC);
```

Comment:
depending on what you intend on using this index for you can possibly combine these into a single index by reordering to
CREATE INDEX runs_v3_idx_ts_epoch_desc_flow_id ON runs_v3 (ts_epoch DESC, flow_id);


Note an index already exists on flow_id